src/Core/Services/SyncService.cs

```diff
@@ -274,6 +274,78 @@ public async Task<bool> SyncDeleteCipherAsync(SyncCipherNotification notificatio
             return SyncCompleted(false);
         }
 
+        public async Task<bool> SyncUpsertSendAsync(SyncSendNotification notification, bool isEdit)
+        {
+            SyncStarted();
+            if (await _stateService.IsAuthenticatedAsync())
+            {
+                try
+                {
+                    var shouldUpdate = true;
```

Comment:
This `shouldUpdate` isn't really necessary, we can just `return SyncCompleted(false);` when we don't need to update the code also reducing the number of ifs in this method

Reply:
I simplified this in 743bf06b24514459a236efd43fa9e297948ed8f9. Let me know if this addresses your change request :)